internal/engine/netx/quicdialer/system_test.go

```diff
@@ -3,15 +3,42 @@ package quicdialer_test
 import (
 	"context"
 	"crypto/tls"
+	"errors"
+	"net"
 	"testing"
 
 	"github.com/lucas-clemente/quic-go"
 	"github.com/ooni/probe-cli/v3/internal/engine/netx/quicdialer"
 	"github.com/ooni/probe-cli/v3/internal/engine/netx/trace"
 	"github.com/ooni/probe-cli/v3/internal/errorsx"
 	"github.com/ooni/probe-cli/v3/internal/netxlite"
+	"github.com/ooni/probe-cli/v3/internal/netxmocks"
+	"github.com/ooni/probe-cli/v3/internal/quicx"
 )
 
+func TestQUICListenerSaverCannotListen(t *testing.T) {
```

Comment:
This test was missing. It's not logically part of this PR, but it feels good to improve coverage.